src/components/movie/DeleteMovieForm.js

```diff
@@ -19,4 +21,11 @@ const DeleteMovieForm = () => {
     return <Form config={deleteFormConfig}/>
 };
 
-export default DeleteMovieForm;
+const mapState = (state) => {
+    return {
+        deleteMovieRequest: state.deleteMovieRequest,
```

Comment:
if it's a function it should be moved to `mapDispatchToProps` and should NOT be a part of store data

Reply:
Fixed